applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -41,7 +41,16 @@
 -define(KEY_DELETE_AFTER_NOTIFY, <<"delete_after_notify">>).
 -define(KEY_SAVE_AFTER_NOTIFY, <<"save_after_notify">>).
 -define(KEY_FORCE_REQUIRE_PIN, <<"force_require_pin">>).
+-define(KEY_ALLOW_FF_RW, <<"allow_ff_rw">>).
+-define(KEY_SEEK_DURATION, <<"seek_duration">>).
 -define(MAX_INVALID_PIN_LOOPS, 3).
+-define(DEFAULT_SEEK_DURATION, 10000).
```

Comment:
In a similar vein, there are time related macros in `kz_types.hrl` to use to ease reading: `10 * ?MILLISECONDS_IN_SECOND` would compile to `10000` but reads a little easier.

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)